### PR TITLE
[RISCV] Re-format RISCVFeatures.td so it doesn't look like AssemblerPredicate is an operand to Predicate.

### PR DESCRIPTION
**Pull request by topperc**
AssemblerPredicate was almost always indented to the same column as the first operand to Predicate. But AssemblerPredicate is a separate base class so should have the same indentation as Predicate.

For the string passed to AssemblePredicate, I aligned it to the other arguments on the previous if it fit in 80 columns. Otherwise I indented 4 spaces past the start of AssemblerPredicate.

For some vendor extensions I put the 2 classes on new lines instead of the same line as the def. This gave more room for the strings and was more consistent with other formatting in that portion of the file.

### CODE REVIEW

File: llvm/lib/Target/RISCV/RISCVFeatures.td [hunk 1]
```diff
@@ -979,11 +986,11 @@ def Feature32Bit
 def Feature64Bit
     : SubtargetFeature<"64bit", "IsRV64", "true", "Implements RV64">;
 def IsRV64 : Predicate<"Subtarget->is64Bit()">,
-                       AssemblerPredicate<(all_of Feature64Bit),
-                                         "RV64I Base Instruction Set">;
+             AssemblerPredicate<(all_of Feature64Bit),
+                               "RV64I Base Instruction Set">;
```
Comment (michaelmaitland) [inline llvm/lib/Target/RISCV/RISCVFeatures.td:990]: Is this line supposed to have two more spaces to be aligned with the start of `all_of`?

Comment (michaelmaitland) [reply llvm/lib/Target/RISCV/RISCVFeatures.td:990]: edit: nevermind it looks good

Comment (michaelmaitland) [reply llvm/lib/Target/RISCV/RISCVFeatures.td:990]: edit: i mean the `(`

**Review by wangpc-pp — APPROVED** [review]
LGTM with nits.

**Review by asb — APPROVED** [review]
Much better, thank you!

**Pull request merged**